constructiveness_toxicity_crowdsource/common/crowd_data_aggregator.py

```diff
@@ -0,0 +1,63 @@
+import pandas as pd
+import numpy as np
+import math
+from crowd_data_aggregation_functions import *
+
+class CrowdsourceAggregator:
+    '''
+    Aggregator for crowdsourced data for constructiveness and toxicity
+    '''
+    def __init__(self, input_csv):
+        self.df = pd.read_csv(input_csv)
+        
+    def get_gold_questions(self):
+        return self.df.query('_golden == True')
+    
+    def get_non_gold_questions(self):
+        return self.df.query('_golden == False')
+            
+    def get_nannotators(self):
+        return len(self.non_gold_df['_worker_id'].unique())
```

Comment:
The class dosn't seem to have an attribute self.non_gold_df. Is this a typo? I think this should call the function self.get_non_god_questions() instead.